std_msgs/Header header

float32 roll		# roll angle [deg]
float32 pitch		# pitch angle [deg]
float32 yaw		# yaw angle [deg]
float32 trottle 	# vertical trottle [m/s]
